src/Owin.Loader/DefaultLoader.cs

```diff
@@ -265,6 +272,11 @@ private Action<IAppBuilder> LoadImplementation(string startupName, IList<string>
             }
             return fullMatch;
         }
+        
+        private static bool DoesTypeNameReferToStartupAttribute(Type type)
```

Comment:
MatchesStartupAttribute

Reply:
> 
> 
> MatchesStartupAttribute
@Tratcher 
Should I rename?

Reply:
Yes, that was a recommendation for a more concise name.